src/lib/Form/Type/Invitation/InvitationType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\User\Form\Type\Invitation;
+
+use Ibexa\Contracts\User\Invitation\InvitationService;
+use Ibexa\User\Form\DataTransformer\InvitationTransformer;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+final class InvitationType extends AbstractType
+{
+    private InvitationService $invitationService;
+
+    public function __construct(InvitationService $invitationService)
+    {
+        $this->invitationService = $invitationService;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->addViewTransformer(new InvitationTransformer($this->invitationService));
```

Comment:
I'm not an expert on Forms, but can we make InvitationTransformer as a service as well and inject that, instead of instantiating it?
Brief look at [the doc](https://symfony.com/doc/5.4/form/data_transformers.html#using-the-transformer) would suggest rather that, unless I missed something.